pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
-]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+[tool.ruff.format]
+quote-style = "single"
```

Comment:
Is there a reason to add ruff here? I assume the whole code base (besides the file modified in this PR) is not formatted with ruff.